docs/methodology.md

```diff
@@ -51,516 +59,118 @@ workloads from the HiBench Benchmark Suite on three Hadoop clusters, each contai
 The dataset is a 258 GiB subset of a larger [14 TiB][hadoop-14TB] dataset (specifically, workers
 1-3).
```

Comment:
Can we report the number of log events in this dataset? The CLP UI should tell you how many there are after you compress.

Reply:
What is CLP UI? Didn't see any input after running `glt c archive/ datasets/`

Reply:
Right, the CLP UI is what you'd see when you use the [CLP package](https://docs.yscope.com/clp/main/user-guide/quick-start-cluster-setup/index.html).